test/DAOUpgradeable.js

```diff
@@ -4,13 +4,19 @@ const TCR = artifacts.require("TracerToken")
 const DAOUpgradeable = artifacts.require("TracerDAO")
 const DAOEmpty = artifacts.require("DAOEmpty")
 const DAOMock = artifacts.require("DAOMock")
+const MultisigDAOUpgradeable = artifacts.require("TracerMultisigDAO")
 const SelfUpgradeableProxy = artifacts.require("CustomUpgradeableProxy")
 const Vesting = artifacts.require("TokenVesting")
+const { deployProxy, upgradeProxy } = require('@openzeppelin/truffle-upgrades');
 
 contract("DAOUpgradable", async (accounts) => {
     const day = 86400
-    const threeDays = 259200
-    const twoDays = 172800
+    const twoDays = day * 2
```

Comment:
I know its not you who originally wrote this but what do we think of time.duration.days(1), days(2) etc

Reply:
Changed